src/main.rs

```diff
@@ -91,3 +89,43 @@ fn print_error_chain(err: anyhow::Error) {
         }
     }
 }
+
+fn run_tests(
```

Comment:
This is quite ugly. I figure we can refactor this as a follow up when we inevitably move more of what's in main into `lib.rs`. Happy to make changes now if anything is really pressing.